presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -0,0 +1,200 @@
+    This is analogous to the (deterministic) :func:`approx_distinct` function.
+
+.. note::
+
+    Unlike :func:`approx_distinct`, this function will return ``NULL`` when ``x`` is empty.
+
+.. function:: noisy_empty_approx_set_sfm(epsilon[, buckets[, precision]]) -> SfmSketch
+
+    Returns an SFM sketch with no items in it. This is analogous to the :func:`empty_approx_set`
+    function, which returns an empty (deterministic) HyperLogLog sketch.
+
+    - ``epsilon`` (double) is a positive number that controls the level of noise in the sketch,
+      as described in [Hehir2023]_. Smaller values of epsilon correspond to noisier sketches.
+    - ``buckets`` (int) defaults to 4096.
+    - ``precision`` (int) defaults to 24.
+
+.. function:: cardinality(SfmSketch) -> bigint
+
+    Returns the estimated cardinality (distinct count) of an ``SfmSketch`` object.
+
+.. function:: merge(SfmSketch) -> SfmSketch
+
+    An aggregator function that returns a merged ``SfmSketch`` of the set union of
+    individual ``SfmSketch`` objects, similar to ``merge(HyperLogLog)``. ::
+
+        SELECT year, cardinality(merge(sketch)) AS annual_distinct_count
+        FROM monthly_sketches
+        GROUP BY 1
+
+.. function:: merge_sfm(ARRAY[SfmSketch, ...]) -> SfmSketch
+
+    A scalar function that returns a merged ``SfmSketch`` of the set union of an array
+    of ``SfmSketch`` objects, similar to :func:`merge_hll`. ::
+
+        SELECT cardinality(merge_sfm(ARRAY[
+            noisy_approx_set_sfm(col_1, 5.0),
+            noisy_approx_set_sfm(col_2, 5.0),
+            noisy_approx_set_sfm(col_3, 5.0)
+        ])) AS distinct_count_over_3_cols
+        FROM my_table
+
+
+Limitations
+-----------
+
+While these functions resemble differential privacy mechanisms, the values returned
+by these functions are not differentially private in general. There are several
+important limitations to keep in mind if using these functions for
+privacy-preserving purposes, including:
+
+- All noisy aggregate functions return ``NULL`` when aggregating empty sets.
+  This means a ``NULL`` return value noiselessly indicates the absence of data.
+- ``GROUP BY`` clauses used in combination with noisy aggregation functions
+  reveal non-noisy information: the presence or absence of a group noiselessly
+  indicates the presence or absence of data. See, e.g., [Wilkins2024]_.
+- Functions relying on floating-point noise may be susceptible to inference
+  attacks such as those identified in [Mironov2012]_ and [Casacuberta2022]_.
+
+---------------------------
+
+.. [Casacuberta2022] Casacuberta, S., Shoemate, M., Vadhan, S., & Wagaman, C.
+    (2022). Widespread Underestimation of Sensitivity in Differentially Private
+    Libraries and How to Fix It. In *Proceedings of the 2022 ACM SIGSAC Conference
+    on Computer and Communications Security* (pp. 471-484).
+
+.. [Hehir2023] Hehir, J., Ting, D., & Cormode, G. (2023). Sketch-Flip-Merge:
```

Comment:
https://arxiv.org/abs/2302.02056

Reply:
I've gone ahead and linked all the references.